<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>materiels</class>
 <widget class="QDialog" name="materiels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
padding: 5px;
border-radius: 20px;
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>731</width>
       <height>451</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
padding: 5px;
border-radius: 20px;
background-color:rgb(188, 74, 60);
font-weight: bold;

}
QPushButton:hover
{
color :white;
}</string>
     </property>
     <property name="title">
      <string>Gestion des matériels</string>
     </property>
     <widget class="QPushButton" name="pushButton_ajou">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
padding: 5px;
border-radius: 20px;
background-color:rgb(188, 74, 60);
font-weight: bold;
}
QPushButton:hover
{
color :yellow;
}</string>
      </property>
      <property name="text">
       <string>Ajouter matériels</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_affi">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>260</y>
        <width>261</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
padding: 5px;
border-radius: 20px;
background-color:rgb(188, 74, 60);
font-weight: bold;
}
QPushButton:hover
{
color :yellow;
}</string>
      </property>
      <property name="text">
       <string>Afficher matériels</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>691</width>
       <height>421</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">groupBox_2{
background-size:contain;
border-image:url(&quot;:/ressources/imgs/5.png&quot;);
background-repeat:no-repeat;
}</string>
     </property>
     <property name="title">
      <string>Ajouter matériels</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Réf:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>NOM:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Quantité:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix TTC:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Disponibilité:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ref">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nom">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_aj2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>240</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
padding: 5px;
border-radius: 20px;
background-color:rgb(188, 74, 60);
font-weight: bold;
}
QPushButton:hover
{
color :yellow;
}</string>
      </property>
      <property name="text">
       <string>Ajouter </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_bk2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
padding: 5px;
border-radius: 20px;
background-color:rgb(188, 74, 60);
font-weight: bold;
border-image:url(&quot;&quot;);
}
QPushButton:hover
{
color :white;
}</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="SpinBox_Qt">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix HT</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="DoubleS_Prht">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>140</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="DoubleS_Prttc">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_affi2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>240</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
padding: 5px;
border-radius: 20px;
background-color:rgb(188, 74, 60);
font-weight: bold;
}
QPushButton:hover
{
color :yellow;
}</string>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_dispo">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>119</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}

QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                 stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
}

/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}

QComboBox:on { /* shift the text when the popup opens */
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow {
    image: url(/usr/share/icons/crystalsvg/16x16/actions/1downarrow.png);
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}</string>
      </property>
      <item>
       <property name="text">
        <string>En Stock</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hors Stock</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_waiting">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>210</y>
        <width>91</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_click">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>220</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_backr">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>10</y>
        <width>671</width>
        <height>341</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>label_backr</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
     <zorder>lineEdit_ref</zorder>
     <zorder>lineEdit_nom</zorder>
     <zorder>pushButton_aj2</zorder>
     <zorder>pushButton_bk2</zorder>
     <zorder>SpinBox_Qt</zorder>
     <zorder>label_4</zorder>
     <zorder>DoubleS_Prht</zorder>
     <zorder>DoubleS_Prttc</zorder>
     <zorder>pushButton_affi2</zorder>
     <zorder>comboBox_dispo</zorder>
     <zorder>label_waiting</zorder>
     <zorder>label_click</zorder>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <property name="styleSheet">
     <string notr="true">border-image:(&quot;:url/imgs/5.png&quot;);</string>
    </property>
    <widget class="QPushButton" name="pushButton_reche">
     <property name="geometry">
      <rect>
       <x>700</x>
       <y>140</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
padding:5px;
border-radius:5px;
background-color:rgb(188, 74, 60);
font-weight:bold;
}
QPushButton:hover
{
color:white;
}</string>
     </property>
     <property name="text">
      <string>Recherche</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ref">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>140</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Réf/Nom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ref_nom">
     <property name="geometry">
      <rect>
       <x>612</x>
       <y>140</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QFrame" name="frame_6">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>210</y>
       <width>251</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(178,184,185);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <widget class="QPushButton" name="pushButton_mod">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
padding:5px;
border-radius:5px;
background-color:rgb(188, 74, 60);
font-weight:bold;
}
QPushButton:hover
{
color:white;
}</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix TTC :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Réf:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>NOM:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Quantité :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prix HT:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Disponiblité :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_rf">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nm">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="DoubleS_prttc">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>100</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="DoubleS_prht">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>130</y>
        <width>62</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_Qt">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="RoomList_dispo">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <property name="currentText">
       <string>En Stock</string>
      </property>
      <item>
       <property name="text">
        <string>En Stock</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hors Stock</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_suppr">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
padding:5px;
border-radius:5px;
background-color:rgb(188, 74, 60);
font-weight:bold;
}
QPushButton:hover
{
color:white;
}</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QTableView" name="tabView_Materiels">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>441</width>
       <height>241</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>30</y>
       <width>191</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(178,184,185);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <widget class="QRadioButton" name="radioButton_ref">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
padding: 5px;
border-radius: 5px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Réference</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_nom">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>50</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
padding: 5px;
border-radius: 5px;

font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_73">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Trier selon :</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_bkaf">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>290</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
padding: 5px;
border-radius: 20px;
background-color:rgb(188, 74, 60);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_chart">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>370</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
padding: 5px;
border-radius: 20px;
background-color:rgb(188, 74, 60);
font-weight: bold;
}
QPushButton:hover
{
color :white;
}</string>
     </property>
     <property name="text">
      <string>           Statistiques</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_excel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>370</y>
       <width>131</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
padding: 5px;
border-radius: 20px;
background-color:rgb(188, 74, 60);
font-weight: bold;


}
QPushButton:hover
{
color :white;
}</string>
     </property>
     <property name="text">
      <string>           Excel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_excel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>370</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*border-image:url(&quot;:/ressources/imgs/6.png&quot;);*/
border-radius:20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_chart">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>370</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/*border-image:url(&quot;:/ressources/imgs/6.png&quot;);*/
border-radius:20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_notific">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_notification">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
padding: 5px;
border-radius: 20px;
background-color:rgb(188, 74, 60);
font-weight: bold;


}
QPushButton:hover
{
color :white;
}</string>
     </property>
     <property name="text">
      <string>        Notification</string>
     </property>
    </widget>
    <zorder>pushButton_notification</zorder>
    <zorder>pushButton_reche</zorder>
    <zorder>label_ref</zorder>
    <zorder>lineEdit_ref_nom</zorder>
    <zorder>frame_6</zorder>
    <zorder>tabView_Materiels</zorder>
    <zorder>frame_5</zorder>
    <zorder>pushButton_bkaf</zorder>
    <zorder>pushButton_chart</zorder>
    <zorder>pushButton_excel</zorder>
    <zorder>label_excel</zorder>
    <zorder>label_chart</zorder>
    <zorder>label_notific</zorder>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QFrame" name="horizontalFrame">
     <property name="geometry">
      <rect>
       <x>-71</x>
       <y>-10</y>
       <width>821</width>
       <height>471</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
